gnomad_qc/v4/resources/basics.py

```diff
@@ -157,14 +182,21 @@ def _ukbb_root_path() -> str:
 
 
 # List of samples to exclude from QC due to withdrawn consents
-# This is the file with the most up to date sample withdrawals we were sent. File downloaded on 8/16/21
```

Comment:
I don't think we need the date downloaded, but I do like having the date in the note 8/9/2021